### PR TITLE
[WIP] Reformatting of code

### PR DESCRIPTION
**Pull request by stephenc**
Follow-up to #1645 and #1654 

@reviewbybees

### CODE REVIEW

**Comment by jglick** [comment]
[WS-ignoring diff](https://github.com/stephenc/jenkins/commit/60e8d23bf2fe42f6781580ebec1b553351e59136?w=1)

File: core/src/main/java/hudson/model/Queue.java [hunk 1]
```diff
@@ -1383,12 +1387,13 @@ public void maintain() {
                 // of isBuildBlocked(p) will become a bottleneck before updateSnapshot() will. Additionally
                 // since the snapshot itself only ever has at most one reference originating outside of the stack
                 // it should remain in the eden space and thus be cheap to GC.
-                // See https://issues.jenkins-ci.org/browse/JENKINS-27708?focusedCommentId=225819&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-225819
+                // See https://issues.jenkins-ci.org/browse/JENKINS-27708?focusedCommentId=225819&page=com.atlassian
+                // .jira.plugin.system.issuetabpanels:comment-tabpanel#comment-225819
```
Comment (jglick) [inline core/src/main/java/hudson/model/Queue.java]: broke up link

**Comment by oleg-nenashev** [comment]
Do we really need it? Refactoring is good, but there're probable merge issues

**Comment by stephenc** [comment]
@oleg-nenashev I do have this debate myself... which is why I have kept #1645 minimal... and with the complexity that #1654 needs to add to remove the potentially duplicated call to `updateSnapshot()` even this PR is out of date

**Comment by oleg-nenashev** [comment]
Now it's not mergeable due to many further changes.
@stephenc maybe it's better to just close it right now.

File: core/src/main/java/hudson/model/Queue.java (hunk 1 above)
Comment (KostyaSha) [inline core/src/main/java/hudson/model/Queue.java]: `https://issues.jenkins-ci.org/browse/JENKINS-27708?focusedCommentId=225819` should just work

**Comment by daniel-beck** [comment]
@stephenc Are you still interested in this PR?

**Comment by recena** [comment]
@stephenc Yes, please. We need for like this.

**Comment by stephenc** [comment]
@daniel-beck I'll see about recreating this fresh

**Pull request closed without merging**